src/main/java/org/apache/sling/feature/cpconverter/handlers/XmlConfigurationEntryHandler.java

```diff
@@ -65,13 +74,77 @@ protected void onJcrRootElement(String uri, String localName, String qName, Attr
 
                     if (attributeValue != null && !attributeValue.isEmpty()) {
                         DocViewProperty property = DocViewProperty.parse(attributeQName, attributeValue);
-
+                        Object value = property.values;
+                        List<String> strValues = Arrays.asList(property.values);
+                        switch (property.type) {
+                            case PropertyType.DATE:
```

Comment:
The JCR https://docs.adobe.com/docs/en/spec/javax.jcr/javadocs/jcr-2.0/javax/jcr/PropertyType.html contains more types than explicitly handled here. Are we sure that we cover all the ones we need?

Reply:
@bosschaert I only handle what the sling osgi installer supports and handles - other types wouldn't work in sling:OsgiConfig nodes anyhow. The cnd doesn't constrain it but the other properties woud be handled as String representations in the osgi installer as well.